mantidproject/mantid
# Plot guess curve in mdf interface

Fixes #12921.

**To test**
1. Open the Multi-dataset fit interface.
2. Add a function.
3. Check the Plot Guess check-box above the function browser.
4.  A graph of the function should be seen in the plot widget (in magenta).
5. Zoom, pan the plot. Function graph should be there for all x values.
6. Changes to parameters / attributes should be reflected in the graph.
7. Add some fitting data. Check that all curves are shown.
8. Switch the guess curve on and off.

Try different functions including composite. When testing with Convolution make sure that FixResolution attribute is 'false'.
